src/index.js

```diff
@@ -0,0 +1,94 @@
+import './style.css';
+import './tabs';
+import './page_load';
+import cont from './contact';
+import createMenu from './menu';
+
+const contact = document.getElementById('contactt');
+const home = document.getElementById('homee');
+const menu = document.getElementById('menuu');
+const heading = document.getElementById('heading-container');
+const para = document.getElementById('para-container');
+const btun = document.getElementById('btn-container');
+cont('bg-col-container');
+const contactHeading = document.getElementById('contact-container');
+const contactMedia = document.getElementById('media-container');
+const contactNumber = document.getElementById('number-container');
+const handle = document.getElementById('handles-container');
+createMenu('bg-col-container');
+const menuContainer = document.getElementById('menu-container');
+
+function displayContact() {
+  contactHeading.classList.remove('dis-none');
+  contactMedia.classList.remove('dis-none');
+  contactNumber.classList.remove('dis-none');
+  handle.classList.remove('dis-none');
+}
+
+function hideContact() {
+  if (!contactHeading.classList.contains('dis-none')) {
+    contactHeading.classList.add('dis-none');
+    contactMedia.classList.add('dis-none');
+    contactNumber.classList.add('dis-none');
+    handle.classList.add('dis-none');
+  }
+}
+
+function hideHomeContact() {
+  hideContact();
+  if (!heading.classList.contains('dis-none')) {
+    heading.classList.add('dis-none');
+    para.classList.add('dis-none');
+    btun.classList.add('dis-none');
+  }
+}
+
+function showMenu() {
+  menuContainer.classList.remove('dis-none');
+}
+
+function hideMenu() {
+  if (!menuContainer.classList.contains('dis-none')) {
+    menuContainer.classList.add('dis-none');
+  }
+}
```

Comment:
- [ ] You should be using `ES6` syntax in this project so here you should use an `arrow function` instead of the `function` keyword. 
Screenshot of requirements 👇🏾 
<img width="470" alt="Screen Shot 2021-04-15 at 8 59 37 PM" src="https://user-images.githubusercontent.com/48804258/114923765-7e9f0600-9e2d-11eb-9cdb-a70153518a83.png">